src/ui/components/Badge/styles.scss

```diff
@@ -0,0 +1,35 @@
+@import "~core/css/inc/mixins";
+@import "~ui/css/vars";
+
+@mixin icon($name) {
+  background: url("~amo/img/icons/badge-#{$name}.svg") 50% 50% no-repeat;
+  content: "";
+  display: inline-block;
+  height: 16px;
+  vertical-align: middle;
+  width: 16px;
+}
+
+.Badge {
+  @include font-bold();
+  @include margin-end(10px);
+
+  background-color: $neutral-base-color;
+  border-radius: 15px;
+  display: inline-block;
+  font-size: $font-size-s;
+  line-height: 16px;
```

Comment:
We should just be using numbers for line height, not pixels. (https://developer.mozilla.org/en/docs/Web/CSS/line-height#Prefer_unitless_numbers_for_line-height_values)

It might not matter a tonne in this particular instance but I think it's a good practice. I guess we should have a style it rule...